spacialgaze-plugins/economy.js

```diff
@@ -114,6 +115,15 @@ function rankLadder(title, type, array, prop, group) {
 	return ladderTitle + tableTop + tableRows + tableBottom;
 }
 
+function economyStat() {
```

Comment:
Do we need this function? Were only calling it in one place tbh, might just want to implement it in the comand.